docs/colang-2/examples/test_csl.py

```diff
@@ -485,37 +485,58 @@ async def test_wait_indefinitely():
 
 
 @pytest.mark.asyncio
-async def test_wait_until_done():
+async def test_it_finished():
     colang_code = """
-# COLANG_START: test_wait_until_done
+# COLANG_START: test_it_finished
 import core
 
-flow handle welcome
-  user said "hi"
+flow bot greet
   bot say "hello"
 
+flow test0
```

Comment:
This is a more technical test, I think we should move this into the the test folder.

I would keep here a human firendly test as this goes into the documentation. I think this one is too complex.

Reply:
Updated it